app/uk/gov/hmrc/gform/ApplicationLoader.scala

```diff
@@ -100,6 +101,10 @@ class ApplicationModule(context: Context)
 
   val formMetadaModule = new FormMetadataModule(mongoModule)
 
+  val queryParameterCrypto =
```

Comment:
Is there any reason why we creating this manually and not using `uk.gov.hmrc.crypto.ApplicationCrypto` from `hmrc-crypto`?